### DISCUSSION TITLE
125G memory; command terminated by signal 9

### DISCUSSION THREAD

**Discussion by marcozhuang21** [The-OpenROAD-Project/OpenROAD-flow-scripts#1296] [Q&A]
why peak memory is so big!! i have upgrade it from 32->64->124. 

[NesterovSolve] Iter: 310 overflow: .811805 HPWL: 2826713260
Command terminated by signal 9
Elapsed time: 4:30:13[h:]min:se. (PU time: user 5172.42 sys 985.29 (37%). Peak memory: 130424528KB
make: ***[Makefile:538: results/asap7/c906/base/3 3 place gp.odb] Error 137

**Comment by maliberty** [comment]
There is no information here with which to answer.  How many instances?  How about a test case?

**Reply by marcozhuang21** [reply]
the project is a riscv core : https://github.com/T-head-Semi/openc906
i think the gates is less than 1M counts

**Reply by marcozhuang21** [reply]
i test two platforms , the same question.
(1) i cp designs/nangate45/tinyRocket as c906 and change config.mk and sdc 
(2) i cp designs/asap7 as c906 and change config.mk and sdc 
[open_test.zip](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/files/12224077/open_test.zip)

**Reply by maliberty** [reply]
Please put together a self-contained test case.  'make global_place_issue' will build one.

**Comment by marcozhuang21** [comment]
i build the  test case as follow: 
https://drive.google.com/file/d/1B--Yyo3EOvMHq2ySyXwNk24psujtZEoZ/view?usp=sharing

**Comment by maliberty** [comment]
I don't se any macros in this design.  Do you have any RAMs in your design?

**Comment by maliberty** [comment]
Moved to an issue https://github.com/The-OpenROAD-Project/OpenROAD/issues/3783
